_posts/2017-06-26-http2.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: post
+title: HTTP2 on MDN
+author: Stephanie Hobson
+excerpt_separator: <!--more-->
+---
+
+We enabled HTTP/2 on MDN's CDN.
+
+We didn't do anything to optimize for HTTP/2, we just enabled it.
+
+We're seeing performance improvements.
+
+<!--more-->
+
+Why is this worth a blog post?
```

Comment:
Consider stating the conclusion instead, like "HTTP/2 isn't scary" or "You don't have to be perfect to use HTTP/2"